exporter/awsxrayexporter/internal/translator/segment.go

```diff
@@ -435,6 +438,19 @@ func makeXRayAttributes(attributes map[string]pcommon.Value, resource pcommon.Re
 				if annoVal != nil {
 					annotations[key] = annoVal
 				}
+			} else if strings.HasPrefix(key, awsxray.AWSXraySegmentMetadataAttributePrefix) {
```

Comment:
Linter wants me to use a switch case.

Reply:
well that is an opinion...

Reply:
Yeah don't really like that, but can't ignore the linter.